core/shared/gc/ems_hmu.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2022 Intel Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include "ems_gc_internal.h"
+
+#if BH_ENABLE_GC_VERIFY != 0
+/* Set default value to prefix and suffix*/
+
+/* @hmu should not be NULL and it should have been correctly initilized (except for prefix and suffix part)*/
+/* @tot_size is offered here because hmu_get_size can not be used till now. @tot_size should not be smaller than OBJ_EXTRA_SIZE.*/
+/*  For VO, @tot_size should be equal to object total size.*/
+void hmu_init_prefix_and_suffix(hmu_t *hmu, gc_size_t tot_size, const char *file_name, int line_no)
+{
+	gc_object_prefix_t *prefix = NULL;
+	gc_object_suffix_t *suffix = NULL;
+	gc_uint32 i = 0;
+
+	bh_assert(hmu);
+	bh_assert(hmu_get_ut(hmu) == HMU_JO || hmu_get_ut(hmu) == HMU_VO);
+	bh_assert(tot_size >= OBJ_EXTRA_SIZE);
+	bh_assert(!(tot_size & 7));
+	bh_assert(hmu_get_ut(hmu) != HMU_VO || hmu_get_size(hmu) >= tot_size);
+
+	prefix = (gc_object_prefix_t *)(hmu + 1);
+	suffix = (gc_object_suffix_t *)((gc_uint8*)hmu + tot_size - OBJ_SUFFIX_SIZE);
+	prefix->file_name = file_name;
+	prefix->line_no = line_no;
+	prefix->size = tot_size;
+	for(i = 0;i < GC_OBJECT_PREFIX_PADDING_CNT;i++)
+	{
```

Comment:
Coding style, `{` should be in the end of last line.
Please download clang-format 12.0.0 from https://github.com/llvm/llvm-project/releases/tag/llvmorg-12.0.0, cd <wamr_root_dir>, and run
`clang-format-12 -i --style=file <path_to_the_source_file>` to format the source codes